documentation/topics/development/working-with-llms.md

```diff
@@ -0,0 +1,46 @@
+# Working with LLMs
```

Comment:
If we duplicate this guide we'll have to maintain it in multiple places. Best just to have it in Ash core I think.

Reply:
I can always count on you to be awake to review PRs.  haha

I'll remove it